src/leap/mx/alias_resolver.py

```diff
@@ -64,3 +74,7 @@ class AliasResolverFactory(LEAPPostfixTCPMapServerFactory):
     """
 
     protocol = LEAPPostfixTCPMapAliasServer
+
+    @property
+    def _query_message(self):
+        return "Resolving alias for"
```

Comment:
I'm sorry: who uses `_query_message` is `LEAPPostfixTCPMapServerFactory.get()`, a method from the parent of this class.